v22.1/show-trace.md

```diff
@@ -197,19 +190,76 @@ In this example, we use two terminals concurrently to generate conflicting trans
+- The OpenTelemetry (OTEL) collector, which can in turn route them to other tools. The OTEL collector is a canonical collector, using the OTLP protocol, that can buffer traces and perform some processing on them before exporting them to Jaeger, Zipkin, and other OTLP tools.
+- Jaeger or Zipkin using their native protocols. This is implemented by using the Jaeger and Zipkin dedicated "exporters" from the OTEL SDK.
+
+The following [cluster settings](cluster-settings.html) are supported:
+
+<table>
+<thead><tr><th>Setting</th><th>Type</th><th>Default</th><th>Description</th></tr></thead>
+<tbody>
+<tr><td><code>trace.span_registry.enabled</code></td><td>boolean</td><td><code>true</code></td><td>If set, ongoing traces can be seen at <code>https://&ltui&gt;/#/debug/tracez</code>.</td></tr>
+<tr><td><code>trace.opentelemetry.collector</code></td><td>string</td><td><code></code></td><td>The address of an OpenTelemetry trace collector to receive traces using the OTEL gRPC protocol, as <code>&lt;host&gt;:&lt;port&gt;</code>. If no port is specified, <code>4317</code> is used.</td></tr>
+<tr><td><code>trace.jaeger.agent</code></td><td>string</td><td><code></code></td><td>The address of a Jaeger agent to receive traces using the Jaeger UDP Thrift protocol, as <code>&lt;host&gt;:&lt;port&gt;</code>. If no port is specified, <code>6381</code> is used.</td></tr>
+<tr><td><code>trace.zipkin.collector</code></td><td>string</td><td><code></code></td><td>The address of a Zipkin instance to receive traces, as <code>&lt;host&gt;:&lt;port&gt;</code>. If no port is specified, <code>9411</code> is used.</td></tr>
+</tbody>
+</table>
+
+### Jaeger example
+
+This example shows how to run Jaeger in Docker and configure CockroachDB to route traces to Jaeger.
+
+1. Run Jaeger in Docker:
+
+    ~~~ shell
+    docker run -d --name jaeger -p 6831:6831/udp -p 16686:16686 jaegertracing/all-in-one:latest
+    ~~~
+
+    This runs the latest version of Jaeger, and forwards two ports to the container. `6831` is the trace ingestion port, `16686` is the UI port. By default, Jaeger will store all received traces in memory.
+
+1. Run CockroachDB and set the Jaeger agent configuration:
+
+    ~~~ sql
+    SET CLUSTER SETTING trace.jaeger.agent='localhost:6831'
+    ~~~
+
+1. Go to [http://localhost:16686](http://localhost:16686).
+1. Select the CockroachDB service, and view traces streaming in.
+
+Instead of searching through log messages in an unstructured fashion, the logs are now graphed in a tree format based on how the contexts were passed around. This also traverses machine boundaries so you don't have to look at different flat `.log` files to correlate events.
+
+Jaeger's memory storage works well for small use cases, but can result in out of memory errors when collecting many traces over a long period of time. Jaeger also supports disk-backed local storage using Badger. To use this, start Jaeger by running the following Docker command:
```

Comment:
Is Badger a thing we can link to?  from this context it seems like some kind of BerkeleyDB-type storage thing, yes?